.travis.yml

```diff
@@ -11,13 +11,32 @@ env:
       secure: IHUYMK2spxorl9lUeAbAfT6btuP2qRT615bUnEuUDgYrXf9y1CdQprYWJaygou/+6aWmLL0NnxYSpOxi40bHgMKeUQnTjXVbkkqQ1Tml3cMSsjkBrx7CNUygHvDDzCQCEC6m9uZjUKMZAVzVSWlOQhSMKR7MtdsSvMCrKIgA2pM=
 
 before_install:
+  # these are all for JS linters, uncomment when they are to be used
+  # - convert -version
+  # - gs -v
```

Comment:
Could you uncomment this and the previous line? Those aren't for the JS linters – they're printing the versions of two of our system dependencies, ImageMagick and GhostScript, to be able to see what Travis is running.